packages/extension/src/hooks/useFees/useFees.test.ts

```diff
@@ -0,0 +1,148 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { Transaction } from 'xrpl';
+
+import { useFees } from './useFees';
+
+// Mock the modules and hooks that `useFees` depends on
+jest.mock('@sentry/react', () => ({
+  captureException: jest.fn()
+}));
+
+let mockGetXrpBalance = () => Promise.resolve('50');
+jest.mock('../../contexts', () => {
+  return {
+    useLedger: () => ({
+      estimateNetworkFees: jest.fn().mockResolvedValue('12'),
+      getAccountInfo: jest.fn().mockResolvedValue({
+        result: { account_data: { OwnerCount: 2 } }
+      })
+    }),
+    useNetwork: () => ({
+      client: { getXrpBalance: mockGetXrpBalance }
+    }),
+    useServer: () => ({
+      serverInfo: {
+        info: { validated_ledger: { reserve_base_xrp: '20' } }
+      }
+    }),
+    useWallet: () => ({ getCurrentWallet: () => ({ publicAddress: 'address' }) })
+  };
+});
+
+describe('useFees', () => {
+  const transaction: Transaction = {
+    TransactionType: 'Payment',
+    Destination: 'fake',
+    Amount: '100000',
+    Account: 'fake',
+    Memos: [
+      {
+        Memo: {
+          MemoData: '54657374206D656D6F',
+          MemoType: '4465736372697074696F6E'
+        }
+      }
+    ]
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should estimate fees correctly', async () => {
+    const { result, waitForNextUpdate } = renderHook(() => useFees(transaction, null));
+
+    // Wait for effects to run and state to update
+    await waitForNextUpdate();
+
+    // Check the results
+    expect(result.current.estimatedFees).toEqual('12');
+  });
+
+  it('useFees with a list of transactions', async () => {
+    const transactions: Transaction[] = [
+      {
+        TransactionType: 'Payment',
+        Destination: 'fake',
+        Amount: '100000',
+        Account: 'fake',
+        Memos: [
+          {
+            Memo: {
+              MemoData: '54657374206D656D6F',
+              MemoType: '4465736372697074696F6E'
+            }
+          }
+        ],
+        Fee: '199'
+      },
+      {
+        TransactionType: 'Payment',
+        Destination: 'fake',
+        Amount: '100000',
+        Account: 'fake',
+        Memos: [
+          {
+            Memo: {
+              MemoData: '54657374206D656D6F',
+              MemoType: '4465736372697074696F6E'
+            }
+          }
+        ]
+        // This transaction has no Fee field, so estimateNetworkFees will be called for it
+      }
+    ];
+
+    // it('should estimate fees correctly for a list of transactions', async () => {
```

Comment:
Is there a reason this comment is still present?

Reply:
Good catch, done